migration-tool/pom.xml

```diff
@@ -32,10 +32,18 @@
 
     <dependencyManagement>
         <dependencies>
+            <!-- We should not use bom-internal here because it'll cause version conflict -->
```

Comment:
Can we align on a single version?

Reply:
Yeah, all the dependent SDK modules use the same version as this module, or do you mean something else?

Reply:
I guess I don't understand why bom-internal would cause version conflicts, then.

Reply:
Some of the dependencies such as jackson and Junit defined in our bom-internal conflict with those defined in `rewrite-recipe-bom`.